config/theme.go

```diff
@@ -84,5 +84,44 @@ func GetColor(key string) tcell.Color {
 func GetColorString(key string) string {
 	themeMu.RLock()
 	defer themeMu.RUnlock()
-	return fmt.Sprintf("#%06x", theme[key].TrueColor().Hex())
+	color := theme[key].TrueColor()
+	if color == tcell.ColorDefault {
+		return "-"
+	}
+	return fmt.Sprintf("#%06x", color.Hex())
+}
+
+// GetReadableColor Returns ColorBlack if the color is brighter than gray
+// otherwise returns ColorWhite
+func GetReadableColor(bg tcell.Color) tcell.Color {
+	if bg == tcell.ColorDefault {
+		return tcell.ColorWhite
+	}
+	r, g, b := bg.RGB()
+	const gray = 0x88 + 0x88 + 0x88 // The sum of gray's R, G, and B
+	if r+g+b > gray {
```

Comment:
The "middle gray" as perceived by humans is 119. So I would set `gray` equal to that. I would calculate the gray value of the color more accurately as `77*r+150*g+29*b + (1<<7) >> 8`, and then compare that to `gray`. Overall this will be more accurate in determining color luminance.

Otherwise looks good, thank you!